Update _index.md (#27844)
DataDog/documentation: content/en/infrastructure/resource_catalog/_index.md

See the [Resource changes collection][8] section of the Google Cloud Platform integration page for instructions on forwarding change events through a Pub/Sub topic and subscription.

> ## Supported Resource types

Reviewer: ```suggestion
## Supported resource types
```

> The Resource Catalog supports 100+ key resource types as listed below. For a full list of resource types supported in the Datadog platform see [here][15].

Reviewer: ```suggestion
The Resource Catalog supports 100+ key resource types. For a full list of resource types supported in the Datadog platform see the [Cloud Resources Schema Reference][15].
```

> 

Reviewer: If this is listed out in the Schema, do you need to list it again here?

[12]: https://docs.datadoghq.com/security/cloud_security_management/identity_risks/
[13]: https://docs.datadoghq.com/security/cloud_security_management/vulnerabilities/
[14]: https://app.datadoghq.com/integrations/azure

> [15]: https://docs.datadoghq.com/infrastructure/resource_catalog/schema/

Reviewer: ```suggestion
[15]: /infrastructure/resource_catalog/schema/
```
